src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const strWithStyles = sourceString.split(';');
+  const objectCss = {};
+
+  strWithStyles.forEach((rule) => {
+    const [key, value] = rule.split(':').map((part) => part.trim());
```

Comment:
Consider chaining the `split` and `trim` operations directly after the `split(';')` for better readability, following the good indentation practices for chained methods.